client.py

```diff
@@ -23,10 +23,12 @@ def send_move(username, move):
 
 
 def get_move(username):
+    print("DEBUG: getting move from server")
     move_url = url + "/game/" + username
     result_raw = requests.get(move_url)
     result = result_raw._content
-    if result[:5] == 'move:':
+    if result[:5] == b'move:':
```

Comment:
What is the purpose of the byte str? Is it neccessary?

Reply:
It's just how the strings come out of the content of the message.  I should probably refactor everything to decode the strings as soon as they come out of the content.